packages/reward-root-submitter/README.md

```diff
@@ -28,42 +28,36 @@ To run test, run
 
 ## Running locally
 
-If you install ganache, you can create a fork of sokol or gnosis chain with
+If you install ganache, you can create a fork of sokol or gnosis with
 
     pdm run fork_sokol
 
 And
 
     pdm run fork_gnosis
 
-Create a .env file with the following contents, adding in the private key and the S3 bucket name.
+A submission can be triggered by pointing to a file either local or on S3
 
-    ETHEREUM_NODE_URL=http://127.0.0.1:8545
-    REWARD_POOL_ADDRESS=0xc9A238Ee71A65554984234DF9721dbdA873F84FA
-    OWNER=0x159ADe032073d930E85f95AbBAB9995110c43C71
-    OWNER_PRIVATE_KEY=
-    REWARD_PROGRAM_OUTPUT=
-    LOGLEVEL=INFO
+    ENVIRONMENT=staging EVM_FULL_NODE_URL=http://127.0.0.1:8545 python -m reward_root_submitter.manual file_path_goes_here
 
-Now running
-
-    python -m reward_root_submitter.main
-
-will enter a 60 second loop that will submit the merkle root for all rewards that have been calculated.
+All required config can be overridden by setting an environment variable, otherwise they will be taken from the secrets manager
 
 ## Development
 
 You can run black to format the code with
 
     pdm run black .
 
-You can build the docker image locally with
+## Manually executing
+
+The lambda can be triggered manually in the AWS UI, and the code can be run from your local machine.
 
-    docker build -t reward_root_submitter .
+Make sure you are in the correct AWS profile for the required environment and run the manual submitter endpoint with a file on S3.
 
-And run the image with
+For example
+
+    ENVIRONMENT=staging python -m reward_root_submitter.manual  s3://cardpay-staging-reward-programs/rewardProgramID=0x5E4E148baae93424B969a0Ea67FF54c315248BbA/paymentCycle=27071744/results.parquet
 
```

Comment:
Can you document information on the event triggers the lambda (in the context of our services)?

Reply:
Added a note on what triggers this service to the readme